src/halmos/sevm.py

```diff
@@ -1348,7 +1356,35 @@ def arith(self, ex: Exec, op: int, w1: Word, w2: Word) -> Word:
             return w1 - w2
 
         if op == EVM.MUL:
-            return w1 * w2
+            if is_bv_value(w1) and is_bv_value(w2):
```

Comment:
can we invoke `is_bv_value(x)` once for each and reuse the results? in the worst case it results in 6 calls to that function, which itself makes more function calls

Reply:
done: https://github.com/a16z/halmos/pull/340/commits/e8490ee2815662b77474a4bcbf5695ac1c119f4e